clients/vscode-hlasmplugin/README.md

```diff
@@ -123,17 +123,22 @@ The following example specifies that GROUP1 is used when working with `source_co
       "program": "second_file",
       "pgroup": "GROUP2"
     },
-  ]
+  ],
+  "alwaysRecognize" : ["*.hlasm", "libs/*.asm"]
 }
 ```
 If you have the two configuration files configured as above and invoke the MAC1 macro from `source_code.hlasm`, the folder `ASMMAC/` in the current workspace is searched for a file with the exact name "MAC1". If that search is unsuccessful the folder `C:/SYS.ASMMAC` is searched. If that search is unsuccessful an error displays that the macro does not exist.
 
-The program field in `pgm_conf.json` supports regular expressions, for example:
+There is also option `alwaysRecognize` which takes array of wildcards. It allows you to configure two things:
+- All files matching these wildcards will be always recognized as HLASM files. 
+- Macros and copy files will be recognized even if their file names does not match the identifier written into source code. For example, macro `IF` would be found even if it was defined in file `IF.hlasm`.
+
+The program field in `pgm_conf.json` supports wildcards, for example:
 ```
 {
   "pgms": [
     {
-      "program": ".*",
+      "program": "*",
```

Comment:
a bit vague example, don't you think?

Reply:
that example should only demonstrate how to get the easiest setup. What do you suggest?

Reply:
maybe something that demonstrates wildcards more, `*.asm`. I agree with your response though, so it's OK for me.

Reply:
common case would probably be a folder with open source files, e.g. pgms/*

Reply:
what about "ASMPGM/*" ?